etc/security/mh_default_org.xml

```diff
@@ -264,11 +265,19 @@
     <sec:intercept-url pattern="/feeds/**" method="GET" access="ROLE_ANONYMOUS" />
 
     <!-- Secure the system management URLs for admins only -->
-    <sec:intercept-url pattern="/services/*" access="ROLE_ADMIN" />
+    <sec:intercept-url pattern="/services/*" method="PUT" access="ROLE_ADMIN" />
```

Comment:
The PUT, POST, and DELETE intercepts here are redundant, and caught by the catch-all on (currently) line 299.